docs/git-integration.md

```diff
@@ -76,3 +76,47 @@ ARGO_WATCHER_DEPLOY_TOKEN=previously_generated_token
 That's it! Starting from this point, Argo-Watcher will be able to commit changes to your GitOps repository.
 
 > Keep in mind, that `argo-watcher` will use the provided tag value as is, without any validation. So, it is up to user to make sure that the tag is valid and can be used for deployment.
+
+## Deployment Lock
+
+Argo-Watcher supports a deployment lock mechanism. It can be useful when you want to prevent Argo-Watcher from making changes during the maintenance window.
+
+There are two ways to enable the deployment lock:
+
+### Scheduled deployment lock
+
+In order to create a scheduled deployment lock, you need to set the following environment variables:
+
+```yaml
+extraEnvs:
+  - name: SCHEDULED_LOCKDOWN_ENABLED
+    value: true
+  - name: LOCKDOWN_SCHEDULE
+    value: '[{"cron": "*/5 * * * *", "duration": "2m30s"}]'
```

Comment:
Can we add human-readable explanations throughout the code, so that it's clear what is the lock interval for this?